src/firefox-patches/disable-workspace-restore.patch

```diff
@@ -0,0 +1,14 @@
+--- a/widget/gtk/nsWindow.cpp
++++ b/widget/gtk/nsWindow.cpp
+@@ -3790,6 +3790,11 @@
+   int32_t workspaceID = workspaceIDStr.ToInteger(&rv);
+ 
+   LOG("nsWindow::MoveToWorkspace() ID %d", workspaceID);
++
++  if (!mozilla::Preferences::GetBool("zen.sessionstore.restore_workspace", true)) {
```

Comment:
This should rather be in https://searchfox.org/firefox-main/source/browser/components/sessionstore/SessionStore.sys.mjs#5737